products/nextjs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Next.js
+category: framework
+tags: javascript-runtime
+iconSlug: nextdotjs
+permalink: /nextjs
+alternate_urls:
+-   /next-js
+-   /nextdotjs
+versionCommand: npx next --version
+releasePolicyLink: https://github.com/vercel/next.js/blob/canary/contributing/repository/release-channels-publishing.md
+changelogTemplate: https://github.com/vercel/next.js/releases/latest
+activeSupportColumn: true
+releaseDateColumn: true
+
+auto:
+-   npm: next
+
+releases:
+-   releaseCycle: "13"
+    releaseDate: 2022-10-25
+    support: true
+    eol: false
+    latest: "13.4.19"
+    latestReleaseDate: 2023-08-19
+-   releaseCycle: "12"
+    releaseDate: 2021-10-26
+    support: false
+    eol: true
```

Comment:
According to https://nextjs-forum.com/post/1140967772285849660 only the latest release is supported. So it's could be fair to assume that a once a new major is out, the previous major becomes EOL.

If this rule is followed this would mean that the `eol` date can be set to the next cycle `releaseDate`, such as eol: 2022-10-25 for 12 (I haven't verified for other release but it seems close to the `latestReleaseDate` for 12).

Reply:
Done, if used the following rule: `EOL(x) = MAX(releaseDate(x+1), latestReleaseDate(x))`